src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -270,6 +309,51 @@ private void GetScopeInformation(StringBuilder builder)
             }
         }
 
+        private async Task ProcessLogQueue()
+        {
+            do
+            {
+                var isExiting = _isExiting;
+                await _semaphore.WaitAsync();
```

Comment:
Added skip; though it should be `IsEmpty` not `Count == 0` as the count test is expensive.